README.md

```diff
@@ -4,52 +4,68 @@ This is a **group exercise**, so you should be working in pairs of two students.
+4. install required libs and packages locally (see requirements.txt)
 
-5. Improve your code based on the review you got. 
+#### For the advanced geoscripting course:
+- activate the corresponding virtual environment
+- run ``conda install -c conda-forge click``
+- run ``pip install --editable .`` (is this actually necessary? can someone try to run it without this step?)
 
 
-## Write an extended random walk program 
+## How to run the tool
+````
+$ random_walker
 
-In this repo you find a basic implementation of a [random walk simulation](https://en.wikipedia.org/wiki/Random_walk) in 2-dimensional space taken from [this blogpost](https://www.geeksforgeeks.org/random-walk-implementation-python/). Running the code yields an image which shows the path of the random walk. 
+Usage: random_walker [OPTIONS] COMMAND [ARGS]...
 
-![random_walk](rand_walk_100000.png)
+Options:
+  --verbose TEXT  Will print verbose messages.
+  --help          Show this message and exit.
 
-The program works but it is not very readable. In addition, you should **extend the program based on the requirements listed below.
+Commands:
+  run  execute command to generate random walkers
+  ````
 
-**Remember to apply the best practices in scientific computing** to make the code more readable, maintainable, reusable and efficient.  
+### How to use the run method
+````
+Usage: random_walker run [OPTIONS]
 
-### Minimum requirements: 
+  execute command to generate random walkers
 
-Extend the program so the following requirements are met:
+Options:
+  -ts, --total_steps INTEGER    Specify the number of total steps for the
+                                random walker, Default is 10,000
 
-1. The program should be able to simulate multiple random walkers. 
-2. The program should be executable from the command line. 
-3. The user should be able to specify the number of random walkers through a command line parameter. 
-4. Document the dependencies and instructions of how to run the program in your README.md.
+  -tw, --total_walkers INTEGER  Specify the number of total walkers, Default
+                                is 1
+
+  -ss, --step_size INTEGER      Specify the size of the steps taken, Default
+                                is 1
 
-### Additional requirements: 
+  -l, --landscape BOOLEAN       Specify whether a grid landscape exists as
+                                base layer or not, Default is False
 
-1. Create three different types of walkers, e.g. a "fast walker" which has a bigger step size. 
-2. Add a "landscape" in which the random walkers are walking in which contains obstacles which the walkers cannot cross (e.g. a lake)
-3. Invent and implement another functionality of your own.
+  -sp, --start_point BOOLEAN    Specify whether the walkers shall start from
+                                the same point or not, Default is False
 
-Be creative here! :) 
+  --help                        Show this message and exit.
 
-## Code Review 
+````
```

Comment:
Really neatly formatted! and really good solution, makes it easy to specify and understand which parameter you're setting